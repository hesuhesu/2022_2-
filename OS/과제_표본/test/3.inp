200
34471 Tristan Bussell
52277 Everett Malinowski
06729 Vikki Bussell
15731 Hannelore Yurick
06523 Troy Reichert
08265 Ezra Dibiase
19126 Shila Gilpin
16203 Collette Burson
44000 Dagny Bencomo
43594 Mackenzie Sloss
04204 Cherie Larsen
08115 Denae Corbeil
28647 Rosenda Yurick
23695 Polly Hagemeier
18127 Sherrie Hagemeier
16810 Arianne Goettl
06388 Sudie Mcglynn
52214 Barbar Gibbens
08351 Ali Parillo
36205 Deloise Blumberg
26466 Tasia Goettl
21409 Corina Yurick
04583 Marlo Nathanson
31373 Rosana Moitoso
16012 Lenore Mcglynn
29220 Gabrielle Nathanson
42904 Sandee Hagemeier
52875 Diana Rubinstein
22253 Dina Hommel
12733 Elois Baize
18380 Miguel Mcglynn
49940 Exie Warshaw
47457 Faviola Burson
13373 Werner Gilpin
44178 Brady Dougherty
01639 Faustino Bussell
37972 Tiffany Gilpin
44217 Darlena Aycock
14203 Kala Churchill
46952 Hertha Nathanson
45520 Robby Batres
04292 Victor Blade
07883 Jesenia Starner
17399 August Row
35072 Danica Rollman
17035 Tatyana Mcglynn
47902 Rosaura Nathanson
23865 Lesha Grasty
35986 Miss Goettl
11369 Genna Bussell
30994 Doyle Yurick
32813 Alda Gilpin
46228 Aracely Saiz
50670 Mabelle Mcgown
12055 Sadie Lovering
22062 Russell Lovering
26515 Jarred Aycock
37450 Franchesca Keplinger
22530 China Taul
46045 Loree Ratliff
05355 Sherika Mcgown
26138 Mui Mcglynn
18844 Tammera Bussell
37020 Margo Youngs
15585 Leann Brunet
06020 Mindy Aycock
52390 Jane Bussell
19927 Kirstie Hagemeier
37797 Rose Stpierre
49217 Mara Tabor
10431 Geneva Wilkinson
36776 Shaunda Mcglynn
06793 Coy Nathanson
18655 Arletta Grosvenor
16860 Ludivina Mcgown
23081 Shon Lovering
01869 Ashley Mcglynn
04658 Amalia Goettl
27342 Jen Yurick
10805 Kristal Illingworth
46778 Hallie Pendergast
23075 Hannah Lovering
00704 Todd Batres
49995 Shaniqua Mcgown
27790 Richie Wingo
44426 Mathew Lovering
03367 Everett Baize
42529 Ignacio Mcglynn
41621 Elmo Fairman
48264 Odilia Goettl
23477 Providencia Yurick
32986 Willodean Lovering
26578 Shanita Convery
29083 Domenic Mcgown
02634 Rodger Lovering
38431 Estell Rollman
20678 Jean Baize
42852 Glinda Mcgown
04960 Bobbi Rollman
39775 Vinnie Rollman
21712 Grisel Pineiro
32404 Rigoberto Bruton
29119 Joeann Womer
12240 Keira Rollman
02143 Xiomara Zielinski
17626 Yer Gilpin
02906 Charlesetta Hommel
35627 Asley Hommel
51075 Christinia Mcgown
40090 Junita Weimar
45946 Marjorie Batres
52224 Charleen Lovering
29891 Elane Rollman
34920 Jacki Burson
15455 Teresita Egli
30525 Lynsey Mcgown
34194 Verline Batres
11247 Larhonda Lovering
16897 Travis Beauchesne
11933 Latasha Baize
44895 Charmain Mcnew
05547 Sherry Yurick
45309 Candelaria Goettl
40460 Gretchen Pflum
33358 Bryant Nathanson
27053 Mari Lore
08486 Kirstie Baize
09455 Thersa Yurick
11755 Shena Rollman
18094 Danika Hagemeier
38848 Jaimee Lovering
31863 Lavonda Nathanson
32062 Magaret Mcgown
46625 Tori Baize
51303 Danille Hambright
21318 Shonna Mcglynn
41249 Kasie Goettl
12822 Tonja Yurick
09154 Jermaine Rollman
01328 Cristina Burson
02416 Julene Hommel
42115 Betsey Nathanson
39540 Rana Hagemeier
12332 Harmony Batres
16565 Delores Pedersen
48599 Cordia Rollman
33645 Bert Yurick
07565 Marcie Hommel
19497 Janette Hollowell
28165 Amberly Nathanson
43176 Lee Goettl
34104 Brandee Jhonson
06753 Nada Ruge
08995 Mariana Gartman
33637 Eleanore Willey
35309 Haley Schlenker
28938 Mignon Aycock
36670 Kimberlie Mcgown
05026 Sonia Mctaggart
13707 Lavona Nathanson
47083 Breanne Mcglynn
24792 Chloe Aycock
09713 Melaine Mcgown
25385 Cody Hagemeier
29593 Tobie Baize
53214 Bettie Helgren
49736 Taylor Mcglynn
08815 Marvis Mcgown
43008 Rona Lovering
03804 Benny Batres
38961 Wonda Traver
26459 Amie Baize
30098 Tricia Goettl
12895 Sylvia Batres
51979 Tiesha Roosa
28090 Toni Lovering
49447 Suzi Mcgown
36324 Kum Christensen
48728 Jaquelyn Lovering
14544 Elza Mcgown
24345 Jenny Mcglynn
37428 Kraig Burson
24997 Fabiola Bussell
09957 Corinna Burson
20275 Karri Goettl
40944 Allegra Aycock
25652 Rikki Wampler
51747 Cristin Burson
15001 Elin Aycock
48131 Kristofer Ulloa
07092 Lynwood Burson
00922 Palma Nathanson
13790 Adah Baize
08210 Jolanda Plain
50288 Dominque Lovering
21084 Eufemia Hagemeier
22987 Jodi Gilpin
26531 Karyn Nathanson
41162 Jude Lovering
39066 Odette Harsh
