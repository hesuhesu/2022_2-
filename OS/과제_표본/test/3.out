00704 Todd        Batres
00922 Palma       Nathanson
01328 Cristina    Burson
01639 Faustino    Bussell
01869 Ashley      Mcglynn
02143 Xiomara     Zielinski
02416 Julene      Hommel
02634 Rodger      Lovering
02906 Charlesetta Hommel
03367 Everett     Baize
03804 Benny       Batres
04204 Cherie      Larsen
04292 Victor      Blade
04583 Marlo       Nathanson
04658 Amalia      Goettl
04960 Bobbi       Rollman
05026 Sonia       Mctaggart
05355 Sherika     Mcgown
05547 Sherry      Yurick
06020 Mindy       Aycock
06388 Sudie       Mcglynn
06523 Troy        Reichert
06729 Vikki       Bussell
06753 Nada        Ruge
06793 Coy         Nathanson
07092 Lynwood     Burson
07565 Marcie      Hommel
07883 Jesenia     Starner
08115 Denae       Corbeil
08210 Jolanda     Plain
08265 Ezra        Dibiase
08351 Ali         Parillo
08486 Kirstie     Baize
08815 Marvis      Mcgown
08995 Mariana     Gartman
09154 Jermaine    Rollman
09455 Thersa      Yurick
09713 Melaine     Mcgown
09957 Corinna     Burson
10431 Geneva      Wilkinson
10805 Kristal     Illingworth
11247 Larhonda    Lovering
11369 Genna       Bussell
11755 Shena       Rollman
11933 Latasha     Baize
12055 Sadie       Lovering
12240 Keira       Rollman
12332 Harmony     Batres
12733 Elois       Baize
12822 Tonja       Yurick
12895 Sylvia      Batres
13373 Werner      Gilpin
13707 Lavona      Nathanson
13790 Adah        Baize
14203 Kala        Churchill
14544 Elza        Mcgown
15001 Elin        Aycock
15455 Teresita    Egli
15585 Leann       Brunet
15731 Hannelore   Yurick
16012 Lenore      Mcglynn
16203 Collette    Burson
16565 Delores     Pedersen
16810 Arianne     Goettl
16860 Ludivina    Mcgown
16897 Travis      Beauchesne
17035 Tatyana     Mcglynn
17399 August      Row
17626 Yer         Gilpin
18094 Danika      Hagemeier
18127 Sherrie     Hagemeier
18380 Miguel      Mcglynn
18655 Arletta     Grosvenor
18844 Tammera     Bussell
19126 Shila       Gilpin
19497 Janette     Hollowell
19927 Kirstie     Hagemeier
20275 Karri       Goettl
20678 Jean        Baize
21084 Eufemia     Hagemeier
21318 Shonna      Mcglynn
21409 Corina      Yurick
21712 Grisel      Pineiro
22062 Russell     Lovering
22253 Dina        Hommel
22530 China       Taul
22987 Jodi        Gilpin
23075 Hannah      Lovering
23081 Shon        Lovering
23477 Providencia Yurick
23695 Polly       Hagemeier
23865 Lesha       Grasty
24345 Jenny       Mcglynn
24792 Chloe       Aycock
24997 Fabiola     Bussell
25385 Cody        Hagemeier
25652 Rikki       Wampler
26138 Mui         Mcglynn
26459 Amie        Baize
26466 Tasia       Goettl
26515 Jarred      Aycock
26531 Karyn       Nathanson
26578 Shanita     Convery
27053 Mari        Lore
27342 Jen         Yurick
27790 Richie      Wingo
28090 Toni        Lovering
28165 Amberly     Nathanson
28647 Rosenda     Yurick
28938 Mignon      Aycock
29083 Domenic     Mcgown
29119 Joeann      Womer
29220 Gabrielle   Nathanson
29593 Tobie       Baize
29891 Elane       Rollman
30098 Tricia      Goettl
30525 Lynsey      Mcgown
30994 Doyle       Yurick
31373 Rosana      Moitoso
31863 Lavonda     Nathanson
32062 Magaret     Mcgown
32404 Rigoberto   Bruton
32813 Alda        Gilpin
32986 Willodean   Lovering
33358 Bryant      Nathanson
33637 Eleanore    Willey
33645 Bert        Yurick
34104 Brandee     Jhonson
34194 Verline     Batres
34471 Tristan     Bussell
34920 Jacki       Burson
35072 Danica      Rollman
35309 Haley       Schlenker
35627 Asley       Hommel
35986 Miss        Goettl
36205 Deloise     Blumberg
36324 Kum         Christensen
36670 Kimberlie   Mcgown
36776 Shaunda     Mcglynn
37020 Margo       Youngs
37428 Kraig       Burson
37450 Franchesca  Keplinger
37797 Rose        Stpierre
37972 Tiffany     Gilpin
38431 Estell      Rollman
38848 Jaimee      Lovering
38961 Wonda       Traver
39066 Odette      Harsh
39540 Rana        Hagemeier
39775 Vinnie      Rollman
40090 Junita      Weimar
40460 Gretchen    Pflum
40944 Allegra     Aycock
41162 Jude        Lovering
41249 Kasie       Goettl
41621 Elmo        Fairman
42115 Betsey      Nathanson
42529 Ignacio     Mcglynn
42852 Glinda      Mcgown
42904 Sandee      Hagemeier
43008 Rona        Lovering
43176 Lee         Goettl
43594 Mackenzie   Sloss
44000 Dagny       Bencomo
44178 Brady       Dougherty
44217 Darlena     Aycock
44426 Mathew      Lovering
44895 Charmain    Mcnew
45309 Candelaria  Goettl
45520 Robby       Batres
45946 Marjorie    Batres
46045 Loree       Ratliff
46228 Aracely     Saiz
46625 Tori        Baize
46778 Hallie      Pendergast
46952 Hertha      Nathanson
47083 Breanne     Mcglynn
47457 Faviola     Burson
47902 Rosaura     Nathanson
48131 Kristofer   Ulloa
48264 Odilia      Goettl
48599 Cordia      Rollman
48728 Jaquelyn    Lovering
49217 Mara        Tabor
49447 Suzi        Mcgown
49736 Taylor      Mcglynn
49940 Exie        Warshaw
49995 Shaniqua    Mcgown
50288 Dominque    Lovering
50670 Mabelle     Mcgown
51075 Christinia  Mcgown
51303 Danille     Hambright
51747 Cristin     Burson
51979 Tiesha      Roosa
52214 Barbar      Gibbens
52224 Charleen    Lovering
52277 Everett     Malinowski
52390 Jane        Bussell
52875 Diana       Rubinstein
53214 Bettie      Helgren

Aycock 7
Baize 9
Batres 7
Burson 8
Bussell 7
Gilpin 6
Goettl 10
Hagemeier 8
Hommel 5
Lovering 15
Mcglynn 12
Mcgown 14
Nathanson 12
Rollman 9
Yurick 10
